### PR TITLE
Added Single branch commit

### PR DESCRIPTION
**Pull request by alwin48**
Added Single Commit function
Screenshot (62)

### CODE REVIEW

**Comment by anandbaburajan** [comment]
> I have created a pull request, i added a single branch commit function. But I think I will soon delete it and work with gitgraph js instead of gitgraph react.

Hmm, any specific reason?

File: src/graph.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+class Graph extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            branches: [],
+            commitMessage:''
+        }
+    }
```
Comment (anandbaburajan) [inline src/graph.js:16]: I'd recommend you to use the newer React's functional components along with hooks. It's easier and simpler to understand and implement, unlike class based components. There are no `class`es, `this`, `componentDidMount`, etc. while using functional components with hooks, so really easy!

File: src/graph.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        this.addCommit();
+    }
+
+    render() {
+        return (
+            <div>
+                <form onSubmit={this.submitHandler}>
+                    <input type="text" value={this.state.commitMessage} onChange={this.changeHandler}/>
+                    <input type="submit" value="Commit on Life"/>
+                </form>
```
Comment (anandbaburajan) [inline src/graph.js:41]: I think the form can be it's own component. The code would be a bit more modular that way. Apart from that, looks good! Thanks for contributing! :D

**Comment by anandbaburajan** [comment]
You can deploy the app already if you wish, at Vercel or Netlify, etc. For example, take a look at https://github.com/FOSS-Cell-GECPKD/image-stego/pull/2. Feel free to get in touch with @BreadGenie if you've any doubts. :-)

**Comment by alwin48** [comment]
Okay thanks for the review @AnandBaburajan

**Comment by alwin48** [comment]
> > I have created a pull request, i added a single branch commit function. But I think I will soon delete it and work with gitgraph js instead of gitgraph react.
> 
> Hmm, any specific reason?

I think gitgraph react is much harder that gitgraph js

**Comment by anandbaburajan** [comment]
@alwin48 I forgot to merge this! Thanks for contributing! :D

**Pull request merged**